templates/gallery/partials/header.snip.html

```diff
@@ -0,0 +1,49 @@
+{{!
+<!---
+Copyright 2017 The AMP Start Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+}}
+
+{{#header}}
+  <header
+    class="
+      gallery-header
+      {{#invert}}gallery-base-invert{{/invert}}
+      {{#mobile-header}}gallery-mobile-header{{/mobile-header}}
+    "
+  >
+    <button
+      class="ampstart-btn gallery-header-menu-button"
```

Comment:
are you going to be using your snipper from previous PR here?

Reply:
No, this is the button to open the sidebar, I actually think this will be removed in the sidebar PR once we refactor the component you suggested -- I'll get back to you on this once I look at the sidebar PR again.